src/modules/notification_events/notification_events.controller.ts

```diff
@@ -29,26 +30,26 @@ export class NotificationEventsController {
   @UseFilters(new AllExceptionsFilter(APIID.TEMPLATE_LIST))
   @Post('/list')
   @ApiBody({ type: SearchFilterDto })
-  @ApiInternalServerErrorResponse({ description: 'Server Error' })
-  @ApiBadRequestResponse({ description: 'Invalid Request' })
+  @ApiInternalServerErrorResponse({ description: ERROR_MESSAGES.INTERNAL_SERVER_ERROR })
+  @ApiBadRequestResponse({ description: ERROR_MESSAGES.INVALID_REQUEST })
   @UsePipes(new ValidationPipe({ transform: true }))
-  @ApiOkResponse({ description: 'Get Template List' })
+  @ApiOkResponse({ description: SUCCESS_MESSAGES.TEMPLATE_LIST })
   async getTemplates(@Body() searchFilterDto: SearchFilterDto, @Res() response: Response) {
-    return this.notificationeventsService.getTemplatesTypesForEvent(searchFilterDto, response)
+    return this.notificationeventsService.getTemplates(searchFilterDto, response)
   }
 
   @UseFilters(new AllExceptionsFilter(APIID.TEMPLATE_GET))
   @Patch("/:id")
   @ApiBody({ type: UpdateEventDto })
-  @ApiResponse({ status: 200, description: "Event updated successfully" })
-  @ApiResponse({ status: 400, description: "Bad request" })
+  @ApiResponse({ status: 200, description: SUCCESS_MESSAGES.UPDATE_TEMPLATE_API })
+  @ApiResponse({ status: 400, description: ERROR_MESSAGES.BAD_REQUEST })
   @UsePipes(new ValidationPipe({ transform: true }))
   updateEvent(
     @Param("id") id: number,
     @Body() updateEventDto: UpdateEventDto,
     @Res() response: Response
   ) {
-    const userId = '016badad-22b0-4566-88e9-aab1b35b1dfc';
+    const userId = '016badad-22b0-4566-88e9-aab1b35b1dfe';
```

Comment:
@poojakarma Why hardcoded userId?

Reply:
mam we will take it from Query param once middleware changes done

Reply:
resolved